backend-start.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-sbt "~backend/reStart"
+sbt -Dsbt.task.timings=true "~backend/reStart"
```

Comment:
I think it should be added to `Publish docker image` step in `/github/workflows/adopt-tapir-ci.yml` as we want to time the tasks during the deployment process.

Reply:
Ok, I will move -Dsbt.task.timings=true from backend-start.sh to adopt-tapir-ci.yml